synadm/api.py

```diff
@@ -154,7 +154,9 @@ def room_delete(self, room_id, new_room_user_id, room_name, message,
     def room_make_admin(self, room_id, user_id):
         """ Make a user into an admin of a room
         """
```

Comment:
Please change the docstring (which is --help in the end) to:

Grant a user the highest power level available in the room. If the user is not in the room, and it is not publicly joinable, then invite the user.

(I hope I got this right, the sentence from the docs is pretty difficult to read (https://github.com/matrix-org/synapse/blob/master/docs/admin_api/rooms.md#make-room-admin-api) and I am not sure if I understand the meaning correctly, so please review if my sentence makes sense here, thanks!)

Reply:
Why did you omit "local" from "...highest power available to a local user..."?

Reply:
i ditched even more from the sentence as I thought it's complicated as it is.

pls make a suggestion i dont get the meaning of it i think

Reply:
```
Grant a user room admin permission. If the user is not in the room, and it is not publicly joinable, then invite the user. 
```

The first short sentence is what will be shown in `synadm room -h` (or even without -h), so that's the most important message for the enduser to be seen.

I don't have a better idea for now. On the very end of my two sentences we could additionally add something that is explaining in detail what exactly room admin permission means. But I don't understand it 100% so let's just leave this out for now and get this finally merged :-) Sorry that this PR took quite some time, but I think it was worth it and thanks again for fixing those nasty bugs! Great additions to the project!

Reply:
Sounds good. Added in 3rd commit.